### PR TITLE
Adds possibility to get meeting invitation by :meeting_id

### PR DESCRIPTION
**Pull request by nickKolya**
Provides endpoint for possibility to get a meeting invitation.
Link to the Zoom API documentation: https://marketplace.zoom.us/docs/api-reference/zoom-api/meetings/meetinginvitation

### CODE REVIEW

**Comment by nickKolya** [comment]
@kyleboe please review

File: lib/zoom/actions/meeting.rb [hunk 1]
```diff
@@ -97,6 +97,16 @@ def livestream(*args)
         options.require(%i[meeting_id stream_url stream_key]).permit(%i[page_url])
         Utils.parse_response self.class.patch("/meetings/#{options[:meeting_id]}/livestream", body: options.except(:meeting_id), headers: request_headers)
       end
+
+      def meeting_invitation(*args)
+        options = Zoom::Params.new(Utils.extract_options!(args))
+        options.require(%i[meeting_id])
```
Comment (kyleboe) [inline lib/zoom/actions/meeting.rb:103]: Minor style adjustment (working on making this consistent across the codebase): Only use symbol array literals for arrays of 2 or more. Thanks!

```suggestion
        options.require(:meeting_id)
```

**Review by kyleboe — CHANGES_REQUESTED** [review]
One small style change. Everything else looks great! Thanks for the contribution and writing tests! 👍

**Review by kyleboe — APPROVED** [review]
Awesome! Thanks for the contribution! 👍

**Comment by nickKolya** [comment]
Hi @kyleboe! Do you have any plans on when this functionality will be put into the next release update?

**Comment by nickKolya** [comment]
> Hi @kyleboe! Do you have any plans on when this functionality will be put into the next release update?

Hey @kyleboe, any update on this? 🙂

**Pull request merged**